provisioning/Installer.ps1

```diff
@@ -29,6 +29,13 @@ if ((Test-Path -Path $installerDir)){
 }
 Invoke-Command $git "clone --branch $($installerBranch) $($installerRepo) $($installerDir)"
 
+# If gpii-hst-tools exists delete it and clone it again
+$hstToolsDir = Join-Path (Join-Path $installerDir "staging") "gpii-hst-tools"
+if ((Test-Path -Path $installerDir)){
```

Comment:
It should be checking if $hstToolsDir exists instead?
And are the double brackets intentional?